workspaces/mend/.changeset/shy-knives-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-mend-backend': patch
+'@backstage-community/plugin-mend': patch
```

Comment:
Because your current version is 0.1.0, this will release a 0.1.1. Is this what you expect?

If you would start with 0.1.0 you should change the package.json version to 0.0.0 and this changeset to minor or major if you want start directly with a 1.0.0.

Reply:
Thanks, updated to release version 0.1.0